README.md

```diff
@@ -1,15 +1,16 @@
-<a href="https://discord.gg/trashdev" target="\_parent">
-  <img alt="Discord" src="https://img.shields.io/discord/796594544980000808?color=7389D8&label&logo=discord&logoColor=ffffff" />
-</a>
-<a href="https://twitter.com/typeheroapp" target="\_parent">
-  <img alt="" src="https://img.shields.io/twitter/follow/typeheroapp.svg?style=social&label=@typeheroapp" />
-</a>
-
-<br />
+<p align=center >
+  <img alt="Github" height=20 src="https://img.shields.io/github/stars/bautistaaa/typehero?style=&logo=github&logoColor=white&label=Stars&labelColor=%23111&color=%23111" />
+  <a href="https://discord.gg/trashdev" target="\_parent">
```

Comment:
Are these `target="\_parent"` intended? (specifically the backslash)

```suggestion
  <a href="https://discord.gg/trashdev" target="_parent">
```

Reply:
these were yanked from tanstack and trpc readmes 🤣

Reply:
We can probably remove it, it needs to be `_parent` exactly